src/qml/AboutPage.qml

```diff
@@ -29,6 +29,28 @@ Flickable {
         id: diskUsage
     }
 
+    Item {
+        id: uptimeCounter
+        property int totaltime : uptime
+
+        Timer {
+            interval: 1000
+            repeat: true
+            triggeredOnStart: true
+            running: true
+            onTriggered: ++uptimeCounter.totaltime
+        }
+        function days() {
+            return Math.floor(uptimeCounter.totaltime / 86400)
+        }
+        function asString() {
+            var now = uptimeCounter.totaltime
+            var days = Math.floor(now / 86400)
```

Comment:
It would help readability if instead of having a 86400 constant there'd be a const secondsPerDay = 24 * 60 * 60 somewhere. This will only be calculated once, it will be more obvious what this does and it could be shared again with the multiplication line 49

Reply:
Done.